packages/venia-concept/src/components/Currency/index.js

```diff
@@ -0,0 +1,38 @@
+import { Component, createElement } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Currency extends Component {
```

Comment:
Were you aware that the [`Price` component](https://github.com/magento-research/pwa-studio/blob/8539d8752fbe2f24efec41d882efed82d06df959/packages/peregrine/src/Price/Price.js) exists? Not clear if you just missed it, if this replaces it, or if this is in addition to it?

Reply:
This is in addition! Not all representations of currency are prices. Subtotals, taxes, difference amounts, etc.

Reply:
I should have looked at the Price component, but I didn't. It looks like the Price component is a better implementation of this; I just think it should be called `Currency`, since not every representation of a currency is a price (e.g. subtotals, difference amounts, etc). I'll switch to this component and separately advocate for its renaming.